pkg/auth/auth_assisted_service_mock_test.go

```diff
@@ -81,71 +84,81 @@ func (f fakeInventory) DownloadMinimalInitrd(ctx context.Context, params install
 }
 
 func (f fakeInventory) DownloadClusterFiles(ctx context.Context, params installer.DownloadClusterFilesParams) middleware.Responder {
-	file, err := ioutil.TempFile("/tmp", "test.file")
+	file, err := ioutil.TempFile("/tmp", "test")
 	if err != nil {
 		return installer.NewDownloadClusterFilesInternalServerError().WithPayload(
 			common.GenerateError(http.StatusInternalServerError, err))
 	}
+
+	defer os.Remove(fmt.Sprintf("{%s}/{%s}", "/tmp", "test"))
+
 	return filemiddleware.NewResponder(
 		installer.NewDownloadClusterFilesOK().WithPayload(io.ReadCloser(file)),
```

Comment:
The file type is already a `ReadCloser`, but also you don't even need a file here. You could use any `ReadCloser` so `io.NopCloser(strings.NewReader("test"))` would work here as well.

This applies to any of these functions that need a `ReadCloser`.

Reply:
Done